frontend/public/index.html

```diff
@@ -1,5 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
```

Comment:
Looks like you accidentally restored a bunch of stuff I'd deleted bc it's boilerplate we're not using for this project. Please remove this.